### ISSUE TITLE
Add new fonts

### ISSUE THREAD

**Issue by Ian-Dx** [CollaboraOnline/richdocumentscode#188]
I'm using truenas scale nextcloud with this great built-in app. It's working great, but it's not scaning any font inside my nextcloud docker /usr/share/fonts, this is a font in my nextcloud docker and in the collabora app it's missing:
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/69476139/175772952-53ea27f9-9f27-41f1-a1c9-498a1c4fb218.png">
so languages like chinese and japanese would be a blank square, any way to fix this?

**Comment by zhimibuwudn** [comment]
English is normal, Chinese box is displayed

**Comment by TobiasReich** [comment]
Would be great, I have the same issue. Chinese fonts (and others) are just empty boxes.

**Comment by pop4ever** [comment]
in the latest nextcloud office app and  builtin CODE server app, we can upload fonts. But how can we make the uploaded fonts to be available in the CODE app?

**Comment by pop4ever** [comment]
Find the solution, install fonts on host machine, and mount font folder of host machine in docker, restart the docker, and it works!

**Comment by snakwu** [comment]
> 

我也碰到这问题了，请问如何操作解决，谢谢

**Comment by joshtrichards** [comment]
https://github.com/CollaboraOnline/richdocumentscode/issues/150#issuecomment-1807125493

The Built-in CODE will poll Nextcloud Office for new fonts once a minute.

**Comment by vFire** [comment]
YES, the script is correct responsed: /apps/richdocuments/settings/fonts.json
And the ttf file is also accessible, but in the opened word files, the characters list is totally different than the one in setting of office, quite weird.
I was noticed that should double check to make coolwsd.xml  setting correctly, but no where to locate this file, seemed is in embed docker image.

**Comment by AndreasMolnar1** [comment]
I have the same issue - does anyone know how to solve it?

**Comment by energygreek** [comment]
when update nextcloud to version 25, this plugin accept fonts in the host!

**Comment by pop4ever** [comment]
> I have the same issue - does anyone know how to solve it?

https://github.com/nextcloud/richdocuments/issues/2763#issuecomment-2189046598

**Comment by joshtrichards** [comment]
This may help: https://help.nextcloud.com/t/what-is-the-difference-between-a-nc-docker-installation-and-bare-metal/215890/3?u=jtr
